pages/common/git-reflog.md

```diff
@@ -0,0 +1,11 @@
+# git reflog
+
+> Show when the refs were updated in local repository.
```

Comment:
Probably good to expand "ref" to references.

Reply:
Done!